launch/launch.go

```diff
@@ -102,9 +104,17 @@ func DefaultSocketPath() string {
 	return socketPath
 }
 
-func mergeEnv(env, jobEnv, optionEnv EnvVar) []EnvVar {
-	for k, v := range jobEnv {
-		env[k] = v
+func mergeEnv(env EnvVar, jobEnv screwdriver.MapSlice, optionEnv EnvVar) []EnvVar {
+	for _, e := range jobEnv.Body {
+		for k, v := range e {
+			val := v
+			for fr, to := range env {
+				val = strings.Replace(val, "${"+fr+"}", to, -1)
```

Comment:
Environment variables can be specified not only as `${FOO}' but also as `$FOO`.

Reply:
I fixed it. I use `os.ExpandEnv`.